compiler/msgs.nim

```diff
@@ -486,6 +486,28 @@ proc formatMsg*(conf: ConfigRef; info: TLineInfo, msg: TMsgKind, arg: string): s
               else: ErrorTitle
   conf.toFileLineCol(info) & " " & title & getMessageStr(msg, arg)
 
+proc colorError(s: string, color: ForegroundColor): string =
+  when not defined(colorfulError):
+    template isQuote(val: untyped): untyped = val == '\''
```

Comment:
and also, should be:
`conf.isDefined("nimColorErrors")` so that you don't need to recompile nim just to toggle this flag